files/containerd-config.toml

```diff
@@ -1,6 +1,7 @@
 version = 2
 root = "/var/lib/containerd"
 state = "/run/containerd"
+imports = ["/etc/containerd/config.d/*"]
```

Comment:
I think we still want the `.toml` filter here, for sanity.

Reply:
i agree, there has been a long debate on temporary files when some folks use utilities that create them some of the k8s config/yaml files

Reply:
gotcha, done!